### ISSUE TITLE
Next.js support regarding this component

### ISSUE THREAD

**Issue by kruserr** [asmyshlyaev177/react-horizontal-scrolling-menu#114]
This may be unrelated as a bug report, since im using nextjs and i am not sure if this component is intended to be used for nextjs.

But here it goes,

I'm using:
```
<ScrollMenu
  ...
  hideArrows={true}
  hideSingleArrow={true}
  ...
/>
```
And the length of the image array is 7.

But auto hiding the left/right arrow when first/last element is visible doesnt work in nextjs.

![image](https://user-images.githubusercontent.com/46799551/92330937-50b93e00-f062-11ea-8d88-b350da5dd09d.png)

I just wanted to hear if fixing this bug for nextjs only, is a interest of the project, i will attempt to fix this for myself, but if i do and there is interest i may do a pr.

**Comment by antoninu** [comment]
Try setting the translate prop to 6, it fixed it for me (I'm also using Next.js)

**Comment by asmyshlyaev177** [comment]
Never tested it with SSR, guess need force component to re-render when it loaded in browser. Maybe add ref to component and call `ref.setInitial()` or `ref.setState({ translate: 6 })` when it mounted in browser.

**Comment by asmyshlyaev177** [comment]
Published version `2.0.0`, it should works.
